docs/reference/patterns/distributed-locks.md

```diff
@@ -31,6 +31,7 @@ already available that can be used for reference.
 * [Redlock-rb](https://github.com/antirez/redlock-rb) (Ruby implementation). There is also a [fork of Redlock-rb](https://github.com/leandromoreira/redlock-rb) that adds a gem for easy distribution.
 * [Redlock-py](https://github.com/SPSCommerce/redlock-py) (Python implementation).
 * [Pottery](https://github.com/brainix/pottery#redlock) (Python implementation).
+* [python-redis-lock](https://github.com/ionelmc/python-redis-lock) (Python implementation).
```

Comment:
It seems to use a slightly different algorithm. I can see that it uses two keys for a lock. I think it's good to describe this, for example:

```suggestion
* [python-redis-lock](https://github.com/ionelmc/python-redis-lock) (Python implementation, a slightly different algorithm).
```

Reply:
@miohtama, thank you for the contribution. Mind accepting the suggestion so we can merge this PR?